<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardFrame</class>
 <widget class="QWidget" name="WizardFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>518</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: white;
}

QFrame
{
	border: 0px;
}

QPushButton
{
	border: 1px solid gray;
	border-radius: 5px;
	padding: 8px;
	color: white;
	background-color: rgba(80, 107, 176);
}

QPushButton:disabled
{
	color: gray;
	background-color: lightgray;
}

QPushButton:pressed 
{
	background-color: rgba(98, 141, 191);
}

QPushButton#btnExit 
{
	color: black;
	padding: 0px;
	border: none; 
	background-color: none;
	width: 70;
}

QLabel#lbStageName, QLabel#lbStage
{
	font: 75 12pt &quot;Trebuchet MS&quot;;
	padding: 0px;
}

QLineEdit 
{
	padding: 8;
	border: 2px solid rgba(160, 175, 211);
	border-radius: 10px;
}

QLineEdit:disabled {
	color: #808080;
	background-color: #d3d3d3;
	border: 2px solid #a6a6a6;
}

QLabel:disabled {
	color: #808080;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="lbStage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">Stage</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbStageName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">Stage name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lbStageHowto">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">Stage howto</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnExit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>#exit</string>
       </property>
       <property name="icon">
        <iconset resource="../../GUI/Resources/Resources.qrc">
         <normaloff>:/Images/button_close.png</normaloff>:/Images/button_close.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="wPage" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wButtonBar" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	border: 1px solid gray;
	border-radius: 5px;
	padding: 8px;
	color: white;
	background-color: rgba(80, 107, 176);
	width: 150px;
}

QPushButton:disabled
{
	color: gray;
	background-color: lightgray;
}

QPushButton:pressed 
{
	background-color: rgba(98, 141, 191);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>60</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>60</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnBack">
        <property name="text">
         <string notr="true">Back</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnForward">
        <property name="text">
         <string notr="true">Forward</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">(C)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../GUI/Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
